docs/src/Upsell.doc.js

```diff
@@ -22,133 +41,370 @@ card(
+                  primaryAction={{href: "https://pinterest.com", label: "Send invite", target: "blank"}}
+                  dismissButton={{
+                    accessibilityLabel: 'Dismiss banner',
+                    onDismiss: ()=>{},
+                  }}
+                  imageData={{
+                    component: <Icon icon="pinterest" accessibilityLabel="" color="darkGray" size={32}/>
+                  }}
+                />
+              </Flex>
+            </Box>
+          </Box>
+        `}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        Keep showing the same Upsell once it has been dismissed. Upsells should only appear a maximum of 2 times to the same user, as they have diminishing returns.
+        `}
+        defaultCode={`
+        <Flex gap={4} direction="column">
+          <Upsell
+            title="Measure ad performance"
+            message="Install the Pinterest tag to track your website traffic, conversions and more."
+            primaryAction={{label: "Install now"}}
+            dismissButton={{
+              accessibilityLabel: 'Dismiss banner',
+              onDismiss: ()=>{},
+            }}
+            imageData={{
+              component: <Icon icon="ads-stats" accessibilityLabel="" color="darkGray" size={32}/>
+            }}
+          />
+          <Upsell
+            title="Measure ad performance"
+            message="Install the Pinterest tag to track your website traffic, conversions and more."
+            primaryAction={{label: "Install now"}}
+            dismissButton={{
+              accessibilityLabel: 'Dismiss banner',
+              onDismiss: ()=>{},
+            }}
+            imageData={{
+              component: <Icon icon="ads-stats" accessibilityLabel="" color="darkGray" size={32}/>
+            }}
+          />
+        </Flex>
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      \`dismissButton\`, \`primaryAction\`, and \`secondaryAction\` each require a short, descriptive label for screen readers, should also be localized.
+
+      In the case of [Buttons](/Button), alternative text should be provided to replace vague text like "Visit" or "Learn more" with more descriptive information, like "Learn more about work from home resources".
+
+      For the \`dismissButton\` [IconButton](/IconButton), the label provided should indicate the intent, like “Dismiss this banner”.
+
+      The [Image](/Image) or [Icon](/Icon) supplied to \`imageData\` should only supply an \`alt\` or \`accessibilityLabel\`, respectively, if the Image or Icon supplies extra context or information. Icons in Upsells are often purely decorative, and can therefore have an empty string as the \`accessibilityLabel\`.
```

Comment:
one of those nits, The Image > Image

Reply:
I think here it makes sense since we are referring to a particular image or icon, and not the component as a whole